libraries/AdaptiveExpressions/BuiltinFunctions/GreaterThan.cs

```diff
@@ -18,14 +19,33 @@ public GreaterThan()
             : base(
                   ExpressionType.GreaterThan,
                   Function,
-                  FunctionUtils.ValidateBinaryNumberOrString,
-                  FunctionUtils.VerifyNumberOrString)
+                  FunctionUtils.ValidateBinary,
+                  FunctionUtils.VerifyNumberOrStringOrDateTime)
```

Comment:
There is no number/string/datetime limitation on the parameters now.

Reply:
fixed